.gitignore

```diff
@@ -115,6 +115,7 @@ website/www/yarn-error.log
 
 # Node
 sdks/typescript/node_modules
+node_modules
```

Comment:
```suggestion
node_modules/**
```

Reply:
Add this then run `git rm --cached -r node_modules` and commit the change. This will remove node_modules files from the branch.

Reply:
Thank you. Removed.